### DISCUSSION TITLE
State of viosock?

### DISCUSSION THREAD

**Discussion by foxtrotcz** [virtio-win/kvm-guest-drivers-windows#1129] [Q&A]
Hello,
I just want to ask what is the state of viosock driver?
Any idea when it will be ready and included in the [Fedora ISO release](https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/archive-virtio/)?
Thanks.

**Comment by YanVugenfirer** [comment]
Hi @FoxtrotCZ , currently you can compile and use it.

Let me discuss internally the plans for its distribution

**Reply by foxtrotcz** [reply]
Thanks, I will wait for the official distribution plans.

**Comment by foxtrotcz** [comment]
Hi, any news about this @YanVugenfirer ?
Maybe @vrozenfe knows if it will be included in ISO distribution?
Thanks.
